src/UI/Components/Diagram/Canvas.test.tsx

```diff
@@ -41,6 +38,7 @@ const user = userEvent.setup();
 const screen = baseWithin(document.body, allQueries);
 
 const setup = (
+  mainService: ServiceModel,
   instance?: InstanceWithRelations,
   serviceModels: ServiceModel[] = services as unknown as ServiceModel[],
```

Comment:
i know this is older code, but perhaps this would be avoided if we used an object in the mock instead of plain json. Can you make a ticket for this? It would be type safer, and avoids the need of bypassing the TS linter.

Reply:
https://github.com/inmanta/web-console/issues/6010